### DISCUSSION TITLE
What does his error mean? Error setting up CodeFormer.

### DISCUSSION THREAD

**Discussion by Timmermanzzz** [TheLastBen/fast-stable-diffusion#1531] [Q&A] [answered]
Everything seems to work but my model isnt showing when I generate images.

**Comment by TheLastBen** [comment, marked as answer]
That's an issue with the dreambooth testing cell, I'll fix it shortly

**Reply by Timmermanzzz** [reply]
Could this be the reason that my trained model isn't showing in stable diffusion?

**Reply by TheLastBen** [reply]
I don't think so
